lib/shoryuken/manager.rb

```diff
@@ -12,7 +13,7 @@ class Manager
 
     def initialize(condvar)
       @count  = Shoryuken.options[:concurrency] || 25
-      @queues = Shoryuken.queues.dup.uniq
+      @polling_strategy = Shoryuken.options[:polling_strategy].new(Shoryuken.queues)
```

Comment:
In my opinion calling it `WeightedRoundRobinStrategy` adds nothing over it being called `WeightedRoundRobin`.